ushahidi/platform-client-mzima
# [Outreachy Task Submission] Fixed Language Accessibility for Japanese on Default Page of the Platform

# Engineers Checklist

## What this PR does?
- Enhances the platforms accessibility for the japanese users by translating all the english words that were there on the default page to japanese.
- This ensures that Japanese users can understand and navigate the platform more effectively in their native language.

## How can it be tested?
- Launch the local server: http://localhost:4200
- Login/signup to the platform
- Navigate to the language feature/section
- Choose Japanese as the language of choice
- Lands to the default page of the platform (which is the Map View)
- Check through to ensure all words and features are in Japanese language


# Where is it documented?
- It is documented on an issue I raised: [The Issue](https://github.com/ushahidi/platform/issues/4828)

## What would be the Impact of the solution?
-The translation of the platform into Japanese will enable Japanese users to seamlessly search, navigate the map, access activity and data views, and utilize collections, resulting in an improved and more inclusive user experience.


## Before the Changes

![Group 3 (5)](https://github.com/ushahidi/platform-client-mzima/assets/124133577/04103b23-072e-4607-9ba3-49315cb4b939)

## After UI Update

![Screenshot from 2024-03-16 18-25-37](https://github.com/ushahidi/platform-client-mzima/assets/124133577/27e023ec-ef8c-460a-b479-b91cbb93945c)

## Steps to achieve this solution
- I took my time doing a lot of research on language translation and used tools like Sider.ai and Microsoft Translator to make the word translations clearer and more accurate. I then implemented these translations based on my findings.
